File: proto/src/main/java/ch/rasc/proto/entity/Person.java
package ch.rasc.proto.entity;

import ch.rasc.extclassgenerator.Model;

@Model(value = "Proto.model.Person", readMethod = "personService.read",
		createMethod = "personService.update", updateMethod = "personService.update",
		destroyMethod = "personService.destroy", rootProperty = "records",
		identifier = "negative")
public class Person extends AbstractPersistable {
	private String firstName;
	private String lastName;

	public String getFirstName() {
		return this.firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return this.lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

}

File: proto/src/main/resources/static/app/model/Person.js
Ext.define("Proto.model.Person",
{
  extend : "Ext.data.Model",
  requires : [ "Ext.data.proxy.Direct", "Ext.data.identifier.Negative" ],
  identifier : "negative",
  fields : [ {
    name : "firstName",
    type : "string"
  }, {
    name : "lastName",
    type : "string"
  }, {
    name : "id",
    type : "integer"
  } ],
  proxy : {
    type : "direct",
    api : {
      read : "personService.read",
      create : "personService.update",
      update : "personService.update",
      destroy : "personService.destroy"
    },
    reader : {
      rootProperty : "records"
    },
    writer : {
      writeAllFields : true
    }
  }
});